include/odp/api/spec/classification.h

```diff
@@ -55,6 +55,13 @@ extern "C" {
  * @deprecated Use ODP_PMR_INVALID instead
  */
 
+/**
+ * @def ODP_MARK_ID_INVALID
```

Comment:
Since CoS mark is not an opaque type, I do not think the definition of ODP_MARK_ID_INVALID can be left to the implementation or ABI. An application, even if not interested in the actual value of ODP_MARK_ID_INVALID, needs to know which mark values it _can_ use without clashing with ODP_MARK_ID_INVALID. It gets quite inconvenient if the value can be anything (like 7 in one implementation, 42 in another).

Reply:
OK. How about setting ODP_MARK_ID_INVALID as zero and change the comments accordingly.

Reply:
Yes, defining it as zero in the API itself would solve this.